JS/blogForm.js

```diff
@@ -28,3 +62,101 @@ let resNavMenuItem = document.getElementsByClassName("res_menu_item")
 for (let i = 0; i < resNavMenuItem.length; i++) {
     resNavMenuItem[i].addEventListener('click', toggleNavBar);
 }
+
+let toolbar_options = [
+    ['bold', 'italic', 'underline', 'strike'],
+    ['link', 'blockquote', 'code-block', 'image'],
+    [{ list: 'ordered' }, { list: 'bullet' }]
+
+]
+
+let quill = new Quill('#blog_content', {
+    modules: {
+        toolbar: toolbar_options,
+    },
+    theme: "snow",
+    placeholder: 'Write the article body here ...',
+})
+
+
+async function uploadProcess(imageToUpload) {
+
+    const data = new FormData();
+    data.append("file", imageToUpload);
+    data.append("upload_preset", "marketMedia");
+    data.append("cloud_name", "doxc03jzw");
+
+    const Http = new XMLHttpRequest();
+    const url = "https://api.cloudinary.com/v1_1/doxc03jzw/image/upload";
+    Http.open("POST", url);
+    Http.send(data);
+
+    Http.onreadystatechange = (e) => {
+        let res = JSON.parse(Http.responseText);
+        imageStored = res.url;
+    }
+}
+
+imageInput.onchange = event => {
+    const file = event.target.files[0];
+    let image = URL.createObjectURL(file);
+
+    newItem.src = image;
+
+    inner_text.classList.add("hide_element");
+    imageContainer.appendChild(newItem);
+    imageContainer.classList.add("preview-img-container")
+    newItem.classList.add("preview-img");
+    uploadProcess(file);
+}
+
+form.addEventListener('submit', async (e) => {
+    e.preventDefault();
+    if (form.title.value.trim() !== '' && form.hook.value.trim() !== '' && form.content.value.trim() !== '' && imageStored !== '') {
+
+        let today = new Date()
+        let today_date = today.getDate() + " " + today.toLocaleString('default', { month: 'short' }) + " " + today.getFullYear()
```

Comment:
Here also it would be good practice to use camelCase but also use temple string instead of trying to add strings 
```javascript

let todaDate = `${today.getDate()}  ${today.toLocaleString('default', { month: 'short' })} .....`;
```